runtime/v2/runc/service.go

```diff
@@ -334,6 +355,19 @@ func (s *service) Start(ctx context.Context, r *taskAPI.StartRequest) (*taskAPI.
 		}
 		s.setCgroup(cg)
 	}
+	if r.ExecID != "" {
+		s.send(&eventstypes.TaskExecStarted{
```

Comment:
You want `sendL` right?

Reply:
No, we are holding the lock a few lines up